### PR TITLE
Add uwu text

### PR DESCRIPTION
**Pull request by gurbaflurb**
Adds UwU text generator, looks like it needs to help merging

### CODE REVIEW

**Comment by gurbaflurb** [comment]
Just fixed merge conflict, not sure why it broke, literally was nothing.

File: cogs/FunAndGames.py [hunk 1]
```diff
@@ -67,5 +67,41 @@ def __init__(self, bot):
...
+            uwuMsg = makeUwUText(self, lastMsg)
+        else:
+            uwuMsg = makeUwUText(self, arg)
+        await ctx.send(uwuMsg)
+            
+    @uwu.error
+    async def uwu_error(self, ctx, error):
+        await ctx.send(f"OwO Oh Noes, Wooks wike an ewwow occuwed. Pwease dwon't hwate mwe .·´¯`(>▂<)´¯`·.\n f{error}")
+
+def makeUwUText(self, lastMsg):
```
Comment (DillonJaap) [inline cogs/FunAndGames.py]: indent this?

Comment (DillonJaap) [inline cogs/FunAndGames.py]: you can still make this part of the cog without using the @command rapper, just indent it and change it to "async def"

File: cogs/FunAndGames.py [hunk 2]
```diff
@@ -67,5 +67,41 @@ def __init__(self, bot):
...
+            uwuMsg = uwuMsg+'w'
+        elif(letter is 'm' or letter is 'M'):
+            uwuMsg = uwuMsg+letter+'w'
+        elif(letter is ' '):
+            if(num is 0):
+                uwuMsg = uwuMsg + " X3 "
+            elif(num is 1):
+                uwuMsg = uwuMsg + " \*nuzzles\* "
+            elif(num is 2):
+                uwuMsg = uwuMsg + " "
```
Comment (DillonJaap) [inline cogs/FunAndGames.py]: this does the same thing as line 103 & 104?

File: cogs/FunAndGames.py [hunk 3]
```diff
@@ -67,5 +67,41 @@ def __init__(self, bot):
...
+            elif(num is 3):
+                uwuMsg = uwuMsg + " UwU "
+            elif(num in [i for i in range(4,12)]):
+                uwuMsg = uwuMsg + " "
```
Comment (DillonJaap) [inline cogs/FunAndGames.py]: can't you do?:
```python
elif (num >= 4):
    uwuMsg = uwuMsg + " "
````

File: cogs/FunAndGames.py (hunk 1 above)
Comment (DillonJaap) [reply cogs/FunAndGames.py]: highlighted extra lines by mistake

Comment (gurbaflurb) [reply cogs/FunAndGames.py]: Hmmm, it doesn't seem to like it when I change it like that

**Pull request merged**